research/src/components/file-anatomy.js

```diff
@@ -0,0 +1,24 @@
+/* eslint-disable jsx-a11y/label-has-associated-control */
+import React from 'react'
+import './anatomy.css'
+
+const FileAnatomy = () => {
+  return (
+    <div class="component-anatomy-wrapper">
+      <input type="checkbox" id="show-slots" />
+      <label for="show-slots"> Show slots</label>
+      <div class="component-anatomy">
+        <host name="input">
+          <part name="button">
```

Comment:
somehow I completely missed this - but this part should be:

`file-selector-button`

Per this resolution in relation with the CSSWG: https://github.com/WICG/open-ui/issues/91#issuecomment-639160328